### PR TITLE
fix: added missing entity return from addcomponent

### PR DESCRIPTION
**Pull request by pas-mike**
Hi, I noticed today that the Entity:addComponent doesn't return the entity as it notes in the comment.

Great work btw! tick-knock is a really well written project, it's a pleasure to use.

### CODE REVIEW

File: src/ecs/Entity.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import {getComponentClass, getComponentId} from './ComponentId';
-import {Class} from '../utils/Class';
-import {Signal} from '../utils/Signal';
-import {isTag, Tag} from './Tag';
-import {ILinkedComponent, isLinkedComponent} from './LinkedComponent';
-import {LinkedComponentList} from './LinkedComponentList';
+import { getComponentClass, getComponentId } from './ComponentId';
```
Comment (mayakwd) [inline src/ecs/Entity.ts:1]: Please revert these changes in import formatting, as they are related to your local formatting settings. But irrelevant for the goal of the pull-request.

File: src/ecs/Entity.ts [hunk 2]
```diff
@@ -841,7 +842,7 @@ export class Entity implements ReadonlyEntity {
    * }
    * ```
    */
-  public* getAll<T>(componentClass: Class<T>): Generator<T, void, T | undefined> {
+  public * getAll<T>(componentClass: Class<T>): Generator<T, void, T | undefined> {
```
Comment (mayakwd) [inline src/ecs/Entity.ts:845]: And this one as well.

**Review by mayakwd — CHANGES_REQUESTED** [review]
Good catch! Please exclude code formatting changes and I'll be able to merge it. 👍 
Thank you for your contribution!

**Review by pas-mike — COMMENTED** [review]
Sorry about that, I've removed the formatting.

**Comment by mayakwd** [comment]
Thank you!
